build.sh

```diff
@@ -85,9 +85,15 @@ case nproc in
     ;;
 esac
 
-if [ -f /bin/ninja ]; then
+if [ -f /usr/bin/clang++ ]; then
```

Comment:
Why not use `type` or `which`?

Reply:
And idk why we need to force clang to the main compiler
IMO, clangd should be language-server only, we should use g++ for *nix system